pkg/exec/exec.go

```diff
@@ -25,6 +25,8 @@ import (
 var (
 	//go:embed bin/tsh
 	tsh []byte
+	//go:embed bin/tsh.md5
```

Comment:
Instead of embedding the md5 hash file into the binary, we can set md5Sum string value at build time using -ldflags, something like https://github.com/shubhindia/go-m1temperature/commit/1d0084bb2d611ee73732ae5c90484293e38b5315. Embedding the file will unnecessarily increase the overall binary size. 😁

Reply:
Thanks @shubhindia for suggestion, also we are already using -ldflags for [version](https://github.com/Ridecell/ridectl/blob/f14ec4fabe2845a2ac0fe0b9adf63eda0d9f11e1/Makefile#L17), we can use that for md5 hash too.